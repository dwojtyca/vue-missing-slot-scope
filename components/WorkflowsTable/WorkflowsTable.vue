<template>
  <div class="workflowsTable">
    <Table>
      <template slot="columns"> COLUMNS CONTENT </template>

      <template slot="slotted" slot-scope="anything">
        SLOTTED CONTENT
      </template>
    </Table>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'nuxt-property-decorator';

@Component
export default class WorkflowsTable extends Vue {}
</script>
