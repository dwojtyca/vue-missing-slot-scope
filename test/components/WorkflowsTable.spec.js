import { shallowMount } from '@vue/test-utils';
import WorkflowsTable from '@/components/WorkflowsTable/WorkflowsTable.vue';

describe('WorkflowsTable', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(WorkflowsTable, {
      localVue: global.localVue,
      router: global.router,
      stubs: ['Table']
    });
  });

  afterEach(() => {
    wrapper = undefined;
  });

  test.only('renders correctly', () => {
    // templates with slot-scope are missing in snapshots - why?
    expect(wrapper.element).toMatchSnapshot();
  });
});
