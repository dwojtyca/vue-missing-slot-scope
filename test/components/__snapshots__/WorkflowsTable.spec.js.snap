// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorkflowsTable renders correctly 1`] = `
<div
  class="workflowsTable"
>
  <table-stub>
    <template>
       COLUMNS CONTENT 
    </template>
  </table-stub>
</div>
`;
